pkg/kubernetes/kubernetes.go

```diff
@@ -56,7 +56,7 @@ import (
 	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/common"
 	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/logger"
 	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/types"
-	cnsfilevolumeclientv1alpha1 "sigs.k8s.io/vsphere-csi-driver/pkg/internal/cnsoperator/cnsfilevolumeclient/v1alpha1"
+	cnsoperatorinternalv1alpha1 "sigs.k8s.io/vsphere-csi-driver/pkg/internalapis/cnsoperator"
```

Comment:
`cnsoperatorinternalv1alpha1` ==> `cnsoperatorinternal`.
We don't need v1alpha1, right ?